index.js

```diff
@@ -904,51 +932,41 @@ const configArray = [
       // https://github.com/eslint/eslint/blob/main/docs/src/rules/prefer-promise-reject-errors.md
       'prefer-promise-reject-errors': 'warn',
       // Warn about state variable and setter names which are not
-      // symmetrically named
-      // https://github.com/jsx-eslint/eslint-plugin-react/blob/master/docs/rules/hook-use-state.md
-      'react/hook-use-state': 'warn',
+      // destructured or symmetrically named
+      // https://eslint-react.xyz/docs/rules/naming-convention-use-state
+      'react-naming-convention/use-state': 'warn',
       // Error on missing sandbox attribute on iframes (good
       // security practice)
-      // https://github.com/jsx-eslint/eslint-plugin-react/blob/master/docs/rules/iframe-missing-sandbox.md
-      'react/iframe-missing-sandbox': 'error',
+      // https://eslint-react.xyz/docs/rules/dom-no-missing-iframe-sandbox
+      'react-dom/no-missing-iframe-sandbox': 'error',
       // Warn about unnecessary curly braces around props and
       // string literal children
       // https://github.com/jsx-eslint/eslint-plugin-react/blob/master/docs/rules/jsx-curly-brace-presence.md
+      // TODO: Migrate to https://eslint.style/rules/jsx-curly-brace-presence
       'react/jsx-curly-brace-presence': [
         'warn',
         { props: 'never', children: 'never', propElementValues: 'always' },
       ],
-      // Error on missing or incorrect `key` props in maps in JSX
-      // https://github.com/jsx-eslint/eslint-plugin-react/blob/master/docs/rules/jsx-key.md
-      'react/jsx-key': [
-        'error',
-        {
-          checkFragmentShorthand: true,
-          checkKeyMustBeforeSpread: true,
-          warnOnDuplicates: true,
-        },
-      ],
+      // Error on missing `key` prop in React elements
+      // https://eslint-react.xyz/docs/rules/no-missing-key
+      'react-x/no-missing-key': 'error',
+      // Error on duplicate `key` props in React elements
+      // https://eslint-react.xyz/docs/rules/no-duplicate-key
+      'react-x/no-duplicate-key': 'error',
       // Disallow React being marked as unused
-      // https://github.com/jsx-eslint/eslint-plugin-react/blob/master/docs/rules/jsx-uses-react.md
-      'react/jsx-uses-react': 'warn',
-      // Error on invalid HTML attributes (only `rel` as of March
-      // 2022)
-      // https://github.com/jsx-eslint/eslint-plugin-react/blob/master/docs/rules/no-invalid-html-attribute.md
-      'react/no-invalid-html-attribute': 'error',
```

Comment:
no equivalent in `eslint-plugin-react-x` or `eslint-plugin-react-dom`, and the usefulness of this is low